includes/Clamp/HostCommand.php

```diff
@@ -12,25 +12,31 @@ class HostCommand extends \Clamp\Command
 
     public function executeSet(array $args = array(), array $options = array())
     {
-        if (!shell_exec('grep -R "' . $this->buildParameters($options) . '" /etc/hosts')) {
-            exec('echo "' . $this->buildParameters($options) . '" | sudo tee -a /etc/hosts');
-            exec('dscacheutil -flushcache');
-            $this->writeln('Host set', ConsoleKit\Colors::GREEN);
-        }
-        else {
-            $this->writeln('Host is already set', ConsoleKit\Colors::YELLOW);
+        $host = reset($this->getConfig("$.host.options"));
```

Comment:
Why do you need to `reset` the array pointer? Did you traverse the array before?

Reply:
I had a hard time accessing that value. The key is 127.0.01 and that poses a challenge for JSONPath's dotted search notation. I tried escaping it also. I tried JSONPath's [] notation also... no luck. I found the reset trick after a few hours of googling... not sure why it works.

I think JSONSearch must traverse it at some point.